client/blocks/followers-count/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External Dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Count from 'components/count';
+import { getSelectedSite } from 'state/ui/selectors';
+
+class FollowersCount extends Component {
```

Comment:
Agreed with updating in a follow up PR. It'd be nice to add to Devdocs as a new block and perhaps support `siteId` as a prop so we can use it in a list of sites as well (like the all-sites view of Stats) with selected site being the default.

It'd also be good to reuse it in a few places. It seems the `/people/followers` page is loading the count a bit later; if it used this new block it would be able to display the data sooner.